app/views/votes/_voter.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if current_user %>
 <div class="vote-arrow pull-left"><!-- class: glyphicon ... is from bootstrap.  see: (http://getbootstrap.com/components/) -->
-  <div><%= link_to " ", post_up_vote_path(post), class: 'glyphicon glyphicon-chevron-up', method: :post  %></div>
+  <div><%= link_to " ", post_up_vote_path(post), class: 'glyphicon glyphicon-chevron-up', method: :post, remote: true  %></div>
   <div><strong><%= post.points %></strong></div>
-  <div><%= link_to " ", post_down_vote_path(post), class: 'glyphicon glyphicon-chevron-down', method: :post %></div>
+  <div><%= link_to " ", post_down_vote_path(post), class: 'glyphicon glyphicon-chevron-down', method: :post, remote: true %></div>
 </div>
 <% end %>
```

Comment:
In app/views/votes/down_vote.js.erb, you need to react to the fact the down vote link was clicked. Internally, in the database, the value was updated, but visually, in the browser, nothing changed. Now is your time and place to have the requesting browser update itself through some JS logic:

```diff
+ +$('#post-<%= @post.id %>-vote').html("<strong><%= @post.points %></strong>"); 
```

Here, I am selecting (`$(…)` a specific spot in the "current page", and replacing (`.html(…)`) its content.

I'll let you do the same for the up_vote.js.erb template, and test it out.